source/NVDAObjects/JAB/__init__.py

```diff
@@ -20,65 +20,66 @@
 
 
 JABRolesToNVDARoles={

```

Comment:
If we are reformatting the entire dictionary, we should fix this up

```suggestion
JABRolesToNVDARoles: Dict[str, controlTypes.Role] = {
```